pkg/ccl/spanconfigccl/spanconfigsqltranslatorccl/testdata/tenant/protectedts

```diff
@@ -36,26 +36,26 @@ descs 54
 
 translate database=db
 ----
-/Tenant/10/Table/5{6-7}                    num_replicas=7 num_voters=5 pts=[1 2]
```

Comment:
hmm just noticed that this PR seems to have removed the pts fields that we expect to be populated post translation. Will investigate on monday.

Reply:
The desc IDs needed changing, picked it up in - https://github.com/cockroachdb/cockroach/pull/76606